apps/range-voting/app/components/VoteRow.js

```diff
@@ -101,13 +101,21 @@ class VoteRow extends React.Component {
                 </Bar>
               ))}
             {options.length > 2 && (
-              <ShowMoreText
+              <Badge
+                shape="compact"
+                background={theme.badgeInfoBackground}
+                foreground={theme.badgeInfoForeground}
+                style={{
+                  cursor: 'pointer',
+                  padding: '2px 8px',
+                  pointerEvents: 'auto'
+                  }}                
                 onClick={() => this.setState({ showMore: !showMore })}
               >
                 {showMore
-                  ? 'Show less options...'
-                  : options.length - 2 + ' more options...'}
-              </ShowMoreText>
+                  ? 'Show less...'
+                  : ' + ' + remOptions + ' more'}
```

Comment:
I couldn't get this to work without creating a const, not sure if there's a better way